src/Sarif.Sarifer/AnalyzeProjectCommand.cs

```diff
@@ -79,14 +79,53 @@ private void MenuCommandCallback(object caller, EventArgs args)
                 {
                     // Disable the menu click when we are analysing.
                     SariferPackageCommand.DisableAnalyzeCommands(this.menuCommandService);
-                    List<string> targetFiles = project.GetMemberFiles();
+
+                    List<string> targetFiles = GetFiles(project);
 
                     this.backgroundAnalysisService.AnalyzeAsync(project.FullName, targetFiles, this.cancellationTokenSource.Token)
                         .FileAndForget(FileAndForgetEventName.BackgroundAnalysisFailure);
                 }
             }
         }
 
+        private static List<string> GetFiles(Project project)
```

Comment:
>static List [](start = 16, length = 11)

are these helpers identical between the two modified files? if so, use a single version only? you could leave the helpers in a single file, mark them as internal, and then have the other file call into them. ?

Reply:
i can create a follow up merging those! thank you